pkgs/applications/editors/helix/default.nix

```diff
@@ -14,7 +14,11 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-THzPUVcmboVJHu3rJ6rev3GrkNilZRMlitCx7M1+HBE=";
 
-  nativeBuildInputs = [ git installShellFiles makeWrapper ];
+  nativeBuildInputs = [ git installShellFiles ];
+
+  preBuild = ''
```

Comment:
You can directly use
```nix
env.HELIX_DEFAULT_RUNTIME="${placeholder out}/lib/runtime"
```

Reply:
I replaced the pre build with your suggestion but I think it's supposed to be `"out"` instead of `out`? Tested locally and it seemed to have worked.